packages/import-node-fetch/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@mongosh/import-node-fetch",
+  "version": "0.0.0-dev.0",
+  "description": "Wrapper for ESM module imports",
+  "main": "./lib/index.js",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/mongodb-js/mongosh.git"
+  },
+  "config": {
+    "unsafe-perm": true
+  },
+  "scripts": {
+    "test": "mocha -r \"../../scripts/import-expansions.js\" --timeout 60000 -r ts-node/register \"./{src,lib}/**/*.spec.ts\"",
+    "test-ci": "node ../../scripts/run-if-package-requested.js npm test",
+    "test-coverage": "nyc --no-clean --cwd ../.. --reporter=none npm run test",
+    "test-ci-coverage": "nyc --no-clean --cwd ../.. --reporter=none npm run test-ci",
+    "eslint": "eslint",
+    "lint": "npm run eslint . && npm run prettier -- --check .",
+    "check": "npm run lint && npm run depcheck",
+    "depcheck": "depcheck",
+    "compile": "tsc -p tsconfig.json",
+    "prepublish": "npm run compile",
+    "prettier": "prettier",
+    "reformat": "npm run prettier -- --write . && npm run eslint --fix"
+  },
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "engines": {
+    "node": ">=14.15.1"
+  },
+  "mongosh": {
+    "unitTestsOnly": true
+  },
+  "devDependencies": {
+    "@mongodb-js/eslint-config-mongosh": "^1.0.0",
+    "@mongodb-js/prettier-config-devtools": "^1.0.1",
+    "@mongodb-js/tsconfig-mongosh": "^1.0.0",
+    "depcheck": "^1.4.3",
+    "eslint": "^7.25.0",
+    "prettier": "^2.8.8",
+    "node-fetch": "^3.3.2"
```

Comment:
That's a regular dependency, not a devDependency, right?

Reply:
oh of course 🤦